commit c06c8aeb61708249d8eb0b17a676d16771ea640b
Author: Dennis Brendel <dbrendel@redhat.com>
Date:   Mon Jan 15 09:55:37 2024 +0100

    manual: fix order of arguments of memalign and aligned_alloc (Bug 27547)
    
    On the summary page the order of the function arguments was reversed, but it is
    in correct order in the other places of the manual.
    
    Reviewed-by: Carlos O'Donell <carlos@redhat.com>

diff --git a/manual/memory.texi b/manual/memory.texi
index fb875f4c3c..3710d7ec66 100644
--- a/manual/memory.texi
+++ b/manual/memory.texi
@@ -1502,7 +1502,7 @@ Space}.
 Allocate a block of @var{size} bytes, starting on a page boundary.
 @xref{Aligned Memory Blocks}.
 
-@item void *aligned_alloc (size_t @var{size}, size_t @var{alignment})
+@item void *aligned_alloc (size_t @var{alignment}, size_t @var{size})
 Allocate a block of @var{size} bytes, starting on an address that is a
 multiple of @var{alignment}.  @xref{Aligned Memory Blocks}.
 
@@ -1510,7 +1510,7 @@ multiple of @var{alignment}.  @xref{Aligned Memory Blocks}.
 Allocate a block of @var{size} bytes, starting on an address that is a
 multiple of @var{alignment}.  @xref{Aligned Memory Blocks}.
 
-@item void *memalign (size_t @var{size}, size_t @var{boundary})
+@item void *memalign (size_t @var{boundary}, size_t @var{size})
 Allocate a block of @var{size} bytes, starting on an address that is a
 multiple of @var{boundary}.  @xref{Aligned Memory Blocks}.
 
